src/drivers/gitea_test.go

```diff
@@ -0,0 +1,226 @@
+package drivers_test
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/git-town/git-town/src/drivers"
+	"github.com/stretchr/testify/assert"
+	httpmock "gopkg.in/jarcoal/httpmock.v1"
+)
+
+type mockGiteaConfig struct {
+	codeHostingDriverName string
+	remoteOriginURL       string
+	giteaToken            string
+	configuredHostName    string
+}
+
+type mockGiteaEndpoints struct {
+	root     string
+	orga     string
+	repo     string
+	version  string
+	prBase   string
+	prOpen   string
+	pr1      string
+	pr1Merge string
+}
+
+func newMockGiteaEndpoints() (mge mockGiteaEndpoints) {
+	mge = mockGiteaEndpoints{}
+	mge.root = "https://gitea.com/api/v1"
+	mge.orga = "git-town"
```

Comment:
A lot of these fields are used only once, and are constants. An example is `mge.orga`. Given that this is just simple test data, can you inline as many fields as possible here? The only fields used in tests seem `prOpen`, `pr1Merge`, and `version`.